From: Michael Schutte <michi@debian.org>
Subject: Change misleading kbd_mode message for XLATE mode
Bug-Debian: http://bugs.debian.org/546712
Forwarded: no

The text said “default (ASCII) mode”, which is wrong in two respects:
XLATE isn’t limited to 7-bit ASCII, but works with various 8-bit
charsets, and the XLATE mode hasn’t been the kernel default for a while.
Thanks for the hint go to Samuel Thibault <sthibault@debian.org>.

The usage of kbd_mode together with grep in shell scripts might become
an issue with this commit, though cursory research with
codesearch.google.com shows only the BiCon project
<http://sourceforge.net/projects/bicon/>, which does “kbd_mode | grep -q
-i Unicode” and is thus unaffected.

---
 src/kbd_mode.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: devel/src/kbd_mode.c
===================================================================
--- devel.orig/src/kbd_mode.c	2011-07-22 19:06:53.000000000 +0200
+++ devel/src/kbd_mode.c	2011-07-22 19:07:00.000000000 +0200
@@ -88,7 +88,7 @@
 		printf(_("The keyboard is in mediumraw (keycode) mode\n"));
 		break;
 	      case K_XLATE:
-		printf(_("The keyboard is in the default (ASCII) mode\n"));
+		printf(_("The keyboard is in xlate (8-bit) mode\n"));
 		break;
 	      case K_UNICODE:
 		printf(_("The keyboard is in Unicode (UTF-8) mode\n"));
